pwiz_tools/BiblioSpec/src/SslReader.cpp

```diff
@@ -228,6 +239,56 @@ SslReader::SslReader(BlibBuilder& maker,
     }
   }
 
+  /**
+   * Parses a crosslinked peptide sequence, adds the modifications from the first crosslinked peptide 
+   * to the "mods" vector, and returns the unmodified sequence.
+   * The modifications that get added to the mods vector will include a modification representing the
+   * mass of the crosslinker plus the masses of all of the linked peptides.
+   */
```

Comment:
Would be good documentation to have an example of the crosslinked format this is parsing.